### PR TITLE
Adding the tests

### PR DESCRIPTION
**Pull request by ulasonat**
Tests for
tr.matrix
tr.polyMatrix

### CODE REVIEW

**Comment by ulasonat** [comment]
Thanks.
You mean the documentation of the R language itself,
And in terms of improving the examples, this kind of example was the only type that I could see in test_that files, you mean adding more tests?

File: tests/testthat/test_trace.R [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  # 34  40
+  # 35  41
+  
+  expect_equal(tr.matrix(c), 67)
+})
+
+test_that("tr.polyMatrix", {
+  a <- polyMgen.d(3, 3, rawData=
+                     ch2pn(c("-3 + z^2","2 + 4*z","-z^2",
+                             "1", "2","3 + z",
```
Comment (namezys) [inline tests/testthat/test_trace.R]: style

File: tests/testthat/test_trace.R [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  # 34  40
+  # 35  41
+  
+  expect_equal(tr.matrix(c), 67)
+})
+
+test_that("tr.polyMatrix", {
+  a <- polyMgen.d(3, 3, rawData = ch2pn(c("-3 + z^2","2 + 4*z","-z^2",
+                                          "1", "2","3 + z",
+                                          "2*z","0","2 - 3*z"), "z"), byrow = TRUE, symb = "z")
```
Comment (namezys) [inline tests/testthat/test_trace.R]:
```
a <- polyMgen.d(3, 3, rawData=ch2pn(c("-3 + z^2", "2 + 4*z","-z^2",
                                      "1", "2", "3 + z",
                                      "2*z", "0", "2 - 3*z"), "z"), 
                             byrow=TRUE, symb="z")
```
be carefull about spaces
(preview here is'n perfect)

File: tests/testthat/test_trace.R [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  # 34  40
+  # 35  41
+  
+  expect_equal(tr.matrix(c), 67)
+})
+
+test_that("tr.polyMatrix", {
+  a <- polyMgen.d(3, 3, rawData = ch2pn(c("-3 + z^2", "2 + 4*z", "-z^2",
+                                          "1", "2", "3 + z",
+                                          "2*z", "0", "2 - 3*z"), "z"), byrow = TRUE, symb = "z")
```
Comment (namezys) [inline tests/testthat/test_trace.R]: if "=" is an operator (in condition for example), you have to use spaces
if "=" is used for send parameter by names, you have to write without they

File: tests/testthat/test_trace.R [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  # 33  39
+  # 34  40
+  # 35  41
+  
+  expect_equal(tr.matrix(c), 67)
+})
+
+test_that("tr.polyMatrix", {
+  a <- polyMgen.d(3, 3, rawData=ch2pn(c("-3 + z^2", "2 + 4*z", "-z^2",
+                                          "1", "2", "3 + z",
```
Comment (namezys) [inline tests/testthat/test_trace.R]: :) forgot spaces here

**Comment by namezys** [comment]
you will kill me :)

And no spaces at the end of string

**Comment by ulasonat** [comment]
Deep apologozies, cannot understand exactly what it is that I need to do. Really sorry.
I'll try to understand what you mean with a better mind tomorrow.

**Pull request merged**